src/gen/item_table.yaml

```diff
@@ -0,0 +1,5459 @@
+    potencyB: 0
+- UltraBoots:
+    category: GEAR
+    nameMsg: MSG_ItemName_UltraBoots
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: GearBoots3
+    itemEntity: GearBoots3
+    icon: gear/boots_3
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_ENEMY ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- Hammer:
+    category: GEAR
+    nameMsg: MSG_ItemName_Hammer
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: GearHammer1
+    itemEntity: GearHammer1
+    icon: gear/hammer_1
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_PLAYER ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- SuperHammer:
+    category: GEAR
+    nameMsg: MSG_ItemName_SuperHammer
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: GearHammer2
+    itemEntity: GearHammer2
+    icon: gear/hammer_2
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_ENEMY ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- UltraHammer:
+    category: GEAR
+    nameMsg: MSG_ItemName_UltraHammer
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: GearHammer3
+    itemEntity: GearHammer3
+    icon: gear/hammer_3
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_2 ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- LuckyStar:
+    category: QUEST
+    nameMsg: MSG_ItemName_LuckyStar
+    fullDescMsg: MSG_ItemFullDesc_LuckyStar
+    shortDescMsg: MSG_ItemShortDesc_LuckyStar
+    hudElement: LuckyStar
+    itemEntity: LuckyStar
+    icon: gear/lucky_star
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_2 ]
+    typeFlags: [ ITEM_TYPE_FLAG_KEY, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- Map:
+    category: QUEST
+    nameMsg: MSG_ItemName_Map
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: Present
+    itemEntity: Placeholder
+    icon: key/Gift
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_2 ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- BigMap:
+    category: QUEST
+    nameMsg: MSG_ItemName_BigMap
+    fullDescMsg: MSG_NONE
+    shortDescMsg: MSG_NONE
+    hudElement: Present
+    itemEntity: Placeholder
+    icon: key/Gift
+    sellValue: -1
+    sortValue: 0
+    targetFlags: [ TARGET_FLAG_2 ]
+    typeFlags: [ ITEM_TYPE_FLAG_GEAR, ITEM_TYPE_FLAG_ENTITY_FULLSIZE ]
+    moveID: MOVE_NONE
+    potencyA: 0
+    potencyB: 0
+- FirstDegreeCard:
+    category: QUEST
+    nameMsg: MSG_ItemName_FirstDegreeCard
```

Comment:
Can't most of these be generated using `MSG_ItemName_{{ name }}`?

Reply:
good point! If there are exceptions, you can just leave those in and make the script generate the name according to that pattern if the nameMsg field is omitted

Reply:
In many cases, yes, but I don't think they should necessarily be so, especially for modders. Many of them are missing/left blank and use MSG_NONE, which is also the default value atm if the field is left out.